exporters/otlp/all/src/test/java/io/opentelemetry/exporter/otlp/http/metrics/OtlpHttpMetricExporterBuilderTest.java

```diff
@@ -74,53 +103,61 @@ void setMeterProvider_null() {
   @Test
   void setMeterProvider() {
     when(meterProvider.get(any())).thenReturn(meter);
-    when(meter.counterBuilder(eq("otlp.exporter.seen"))).thenReturn(counterBuilder);
+    when(meter.counterBuilder(any())).thenReturn(counterBuilder);
```

Comment:
ExporterMetrics obtains / increments multiple counters. Only accounting for one of them was causing NPE.